code/modules/mob/living/living_defines.dm

```diff
@@ -64,6 +64,7 @@
 	var/is_ventcrawling = 0
 
 	var/species_type
+	var/mob_species_flags
```

Comment:
I'd rather you renamed this to be more clear about what it's used for, because the current uses aren't really for species.